### ISSUE TITLE
Accept and return values for response_group

### ISSUE THREAD

**Issue by hannahwhy** [NUARIG/nu_surveyor#84]
For each response, Surveyor will send a `response_group` string attribute.  This value needs to be round-tripped.

At present, Surveyor's `response_group` attribute is defined as follows:

```
      Column       |            Type             |                       Modifiers                        
-------------------+-----------------------------+--------------------------------------------------------
 response_group    | character varying(255)      | 
```

**Comment by hannahwhy** [comment]
NUSurveyor also needs to generate values for `response_group` when working with repeating sections, but that's a different issue.

**Comment by jdzak** [comment]
Added in pull request #86 for task #24.

**Comment by yoon** [comment]
closed by 9842a68
